scripts/idl/matter_idl_types.py

```diff
@@ -138,6 +147,11 @@ class Event:
     code: int
     fields: List[Field]
     readacl: AccessPrivilege = AccessPrivilege.VIEW
+    attributes: Set[EventAttribute] = field(default_factory=set)
```

Comment:
This should probably be named "qualities" to match the spec... An event having "attributes" is a bit odd, since  "attribute" is a first order element